pkg/sloop/processing/eventcount_test.go

```diff
@@ -21,9 +21,9 @@ import (
 )
 
 var (
-	someEventWatchTs       = time.Date(2019, 8, 29, 21, 24, 55, 6, time.UTC)
+	someEventWatchTs       = time.Now().UTC()
```

Comment:
Please use deterministic values in unit tests.  Avoid time.Now() as this is going to be a race condition